utils/setup.py

```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name="artemis_utils",
-    version="1.0.1",
+    version="1.0.2",
```

Comment:
this needs to be released as a new version with the changes (init code augmentation plus ARTEMIS license)

Reply:
see https://pypi.org/project/artemis-utils/#history (should be updated)